### PR TITLE
Lpal 878 add 'black'. python formatter to pre-commit hooks, run on all python

### PR DESCRIPTION
**Pull request by nickdavis2001**
## Purpose

_'black' makes python code standards compliant. add this to pre-commit hooks and run it on existing python code_

## Approach

_add 'black'. python formatter to pre-commit hooks, run on all python_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: tests/load/tests/suite.py [hunk 1]
```diff
@@ -92,11 +107,11 @@
         self.client.verify = False
 
         # create the user on the back-end system
-        logging.info(f'creating and activating user: {self.username} / {self.password}')
+        logging.info(f"creating and activating user: {self.username} / {self.password}")
```
[the remark this answers is not in the document]
Comment (williamfalconeruk) [reply tests/load/tests/suite.py:110]: @nickdavis2001  - is this test code?

[the remark this answers is not in the document]
Comment (nickdavis2001) [reply tests/load/tests/suite.py:110]: @williamfalconeruk Yes.  I'm not sure whether its necessary really that it logs the password, so we could take it out anyway to stop it complaining. What you you think?

[the remark this answers is not in the document]
Comment (williamfalconeruk) [reply tests/load/tests/suite.py:110]: go for it!

[the remark this answers is not in the document]
Comment (nickdavis2001) [reply tests/load/tests/suite.py]: Done :)

**Review by townxelliot — APPROVED** [review]

**Pull request merged**